server/tests/controller/article.spec.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable no-console */
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '<server>/app';
+import models from '<server>/models';
+import { article } from '<fixtures>/article';
+import { secondUser } from '<fixtures>/user';
+
+const { expect } = chai;
+chai.use(chaiHttp);
+
+const { User, sequelize } = models;
+
+before(async () => {
+  try {
+    await sequelize.sync({ force: true });
+  } catch (error) {
+    console.log(error);
+  }
+});
+
+describe('/POST create an article', async () => {
+  before(async () => {
+    try {
+      const user = await User.create(secondUser);
+      article.userId = user.id;
+    } catch (error) {
+      console.log(error);
+    }
+  });
+
+  it('should create a new article controller', (done) => {
+    chai
+      .request(app)
+      .post(`/api/v1/articles?userId=${article.userId}`)
+      .send(article)
+      .end((err, res) => {
+        expect(res).to.have.status(201);
+        expect(res.body.status).to.be.equal(201);
+        done();
+      });
+  });
+});
```

Comment:
I also think it will be nice to test for scenarios where the user enters an empty object or invalid fields